packages/tocco-util/src/route/route.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import storeFactory from '../storeFactory'
+import asyncRoute from './asyncRoute'
+import {dispatchInput} from './input'
+
+export const loadRoute = (store, input, importRouteDependencies, path) => props => {
+  const Component = asyncRoute(() =>
+    new Promise(resolve => {
+      importRouteDependencies().then(imported => {
+        const route = imported.default
```

Comment:
A route doesn't necessarily have to contain `reducers` and `sagas` (not sure about reducers, but sagas are clearly optional). I think there should be null checks to avoid null pointers.